pkg/asset/manifests/ingress.go

```diff
@@ -0,0 +1,137 @@
+package manifests
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+
+	configv1 "github.com/openshift/api/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	ingCrdFilename = filepath.Join(manifestDir, "cluster-ingress-01-crd.yml")
+	ingCfgFilename = filepath.Join(manifestDir, "cluster-ingress-02-config.yml")
+)
+
+const (
+	ingressConfigCRD = `
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
There are no variables in this file.  I'd rather follow #592 and move this constant to `data/data/manifests/bootkube/cluster-ingress-02-config.yaml`.

Reply:
Done (I assume you meant `data/data/manifests/bootkube/cluster-ingress-01-crd.yaml`).  Thanks!